modules/jsonld/src/test/scala/ch/epfl/bluebrain/nexus/rdf/jsonld/JsonLdSpec.scala

```diff
@@ -246,5 +319,63 @@ class JsonLdSpec extends RdfSpec {
           .leftValue shouldEqual ContextNotFound(contextUri)
       }
     }
+
+    "return context value" when {
+      "json is an array" in {
+        json"""
+         [
+           {
+             "@context": {
+               "xsd": "http://www.w3.org/2001/XMLSchema#"
+             }
+           },
+           {
+             "@context": {
+               "schema": "http://schema.org/"
+             }
+           }
+         ]
+         """.contextValue shouldEqual
+          json"""
+          {
+            "xsd": "http://www.w3.org/2001/XMLSchema#",
+            "schema": "http://schema.org/"
+          }
+          """
+      }
+    }
+    "remove nested keys" when {
```

Comment:
This is just testing one case, when the field to be removed is inside an array.
The test we had before was testing all the cases:

```json
{
  "@context": "http://example.com/1",
  "field1": {
    "@context": "http://example.com/2",
    "field2": {
      "field3": "Other text"
    }
  },
  "field3": [
    {
      "@context": {
        "xsd": "http://www.w3.org/2001/XMLSchema#"
      }
    },
    {
      "@context": "http://example.com/3",
      "field4": 2
    }
  ],
  "field5": "Some text"
}
```
`json.removeNestedKeys("@context")` and the result should be

```json
{
  "field1" : {
    "field2" : {
      "field3" : "Other text"
    }
  },
  "field3" : [
    {
      "field4" : 2
    }
  ],
  "field5" : "Some text"
}
```